roles/monitoring-common/defaults/main.yml

```diff
@@ -11,4 +11,4 @@ monitoring:
     vhost: /sensu
     username: sensu
     password: sensu
-  cluster_prefix: "ip a | grep :ucarp > /dev/null &&"
\ No newline at end of file
+  cluster_prefix: "ip a | grep :ucarp > /dev/null"
```

Comment:
Maybe we should just write a nice wrapper:
`if_active_controller /path/to/metrics-nova-oversub.sh`